### PR TITLE
`ruff_formatter` crate doc comment fixes

### PR DESCRIPTION
**Pull request by cnpryer**
This PR updates some of the generated docs for the `ruff_formatter` crate. I figure I could throw some of these up as I'm reading through it. 

I'd like to leave this as a draft PR so that I can ask questions with review comments if that's okay.

### CODE REVIEW

File: crates/ruff_formatter/src/macros.rs [hunk 1]
```diff
@@ -116,7 +116,7 @@ macro_rules! dbg_write {
 
 /// Creates the Format IR for a value.
 ///
-/// The first argument `format!` receives is the [`crate::FormatContext`] that specify how elements must be formatted.
+/// The first argument `format!` receives is the [`crate::FormatContext`] that specifies how elements must be formatted.
```
Comment (cnpryer) [inline crates/ruff_formatter/src/macros.rs:119]: This is technically correct both ways. TIL

**Comment by cnpryer** [comment]
Hmm. Yea my questions so far are just specific to the `Printer` implementation. So I'll just throw those on discord and mark this as ready.

File: crates/ruff_formatter/src/macros.rs (hunk 1 above)
Comment (cnpryer) [reply crates/ruff_formatter/src/macros.rs]: 933e8d11967dc9ff5bca9af106f9890edbdd3cbc

**Comment by MichaReiser** [comment]
Thank you

**Pull request merged**